test/jdk/javax/naming/spi/providers/InitialContextTest.java

```diff
@@ -256,22 +280,20 @@ private static void javac(Path compilationOutput, Path... sourceFiles) {
         commands.addAll(paths.stream()
                 .map(Path::toString)
                 .collect(Collectors.toList()));
-        quickFail(run(new ProcessBuilder(commands)));
-    }
-
-    private static void quickFail(Result r) {
-        if (r.exitValue != 0)
-            throw new RuntimeException(r.output);
+        try {
+            OutputAnalyzer outputAnalyzer = ProcessTools.executeCommand(commands.toArray(new String[commands.size()]));
+            if(outputAnalyzer.getExitValue() != 0) {
+                throw new RuntimeException(outputAnalyzer.getOutput());
```

Comment:
Same comment as above `outputAnalyzer.shouldHaveExitValue(0)` would be better.

Reply:
Applies to some other files in this PR as well.

Reply:
Thanks. Implemented in this commit.